sequencers/modeshape-sequencer-ddl/src/test/resources/ddl/create_table_check_constraint.ddl

```diff
@@ -0,0 +1,21 @@
+--  SQL Server / Oracle / MS Access, not named
+--  Note that starting with 5.7 MySQL does not support CHECK constraints 
```

Comment:
well, the MySQL 5.7 documentation states:
> The CHECK clause is parsed but ignored by all storage engines. 

As such, I don't think there's anything MySQL specific the parser should do. i.e. the default `CHECK` handling (which seems to be the same for the other supported DBs) should be enough